### PR TITLE
Add UI config page #20

### PR DESCRIPTION
**Pull request by uzuran**
- add button for settings, select language
- add separate window for settings OptionScreen.py

### CODE REVIEW

**Comment by Nuzhyn-Ivan** [comment]
Please add check - if settings screen opened - do not open another one
image

**Comment by Nuzhyn-Ivan** [comment]
1. Title cut
2. dropdown should not allow to edit values
image

**Comment by Nuzhyn-Ivan** [comment]
Not authorized user can view\change configs. Its not a big deal now, but better to move config to main screen. 
Some use cases also requare to change settings from main screen instead of login screen. For example - user might want to change font size after login, than change it again nad so on

File: Languages.py [hunk 1]
```diff
@@ -1,117 +1,23 @@
...
-# configuration["user_not_exist_warning"] = "Uživatel neexistuje!"
-# configuration["material_exist_warning"] = "Tento materiál již existuje."
-# configuration["material_specification_warning"] = "Nemůžete přidat materiál této specifikace.."
-# configuration["delete_material_warning"] = "Opravdu chcete odstranit tento materiál?"
-#
 
+def change_language(lang):
+    current_lang.clear()
+    config_object = ConfigParser()
+    config_object.read(f"{lang}.ini", encoding="UTF-8")
```
Comment (Nuzhyn-Ivan) [inline Languages.py:20]: lang expected in lowercase here but got first char in upper case. In result - cant parse ini file
Should be something like:
config_object.read(f"{lang.lower()}.ini", encoding="UTF-8")

File: czech.ini [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+dropdown_english = Anglický
+dropdown_czech = Český
+check_who_is_login_label = je  přihlášený.
+login_button = Přihlásit se
+registration_button = Registrace
+registration_label = Registrace nového uživatele
+user_create_button = Registrace
+type_of_material_label = Typ materiálu
+id_label = Id
+thickness_label = Thickness
```
Comment (Nuzhyn-Ivan) [inline czech.ini:21]: Is it correct?

File: czech.ini [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+search_button = Vyhledat materiál
+add_to_storage_button = Přidat materiál do skladu
+frame1 = Železo
+frame2 = Hliník
+frame3 = Nerez
+frame4 = Speciální materiál
+frame5 = Odpis materiálů
+treeview_id = Id:
+treeview_thickness = Tloušťka:
+treeview_x = X size:
```
Comment (Nuzhyn-Ivan) [inline czech.ini:38]: Is it correct?

File: czech.ini (hunk 2 above)
Comment (uzuran) [reply czech.ini:21]: thickness_label = Tloušťka

File: OptionScreen.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        super().__init__(child)
+
+        self.geometry("200x250")
+        self.title(Languages.current_lang["settings_title"])
+
+        # Dropdown menu for options
+        self.drop_down_label = tk.Label(self, text=Languages.current_lang["choice_your_lang"])
+        self.drop_down_label.pack(side="top")
+
+        self.languages = ["English", "Czech"]
```
Comment (Nuzhyn-Ivan) [inline OptionScreen.py:25]: You can avoid hardcode here by using languages dict. Then, when you want to add another language - just add it in one place. Another option here - move languages dict to Languages.py

**Comment by Nuzhyn-Ivan** [comment]
Value in languages dropdown is in a fixed order. Would be nice to see current language selected when open settings
image

**Comment by Nuzhyn-Ivan** [comment]
After change language - title of Login screen stay on previous language
image

**Comment by Nuzhyn-Ivan** [comment]
> Settings screen actually a pop-up, and probably should be modal if possible

For now its impossible to open new popup/window if its(or other) is already opened. 
But its still possible to press controls on 'mother' window so it could be confusin for user. 
Please consider to implement canonical modal behaviour. Maybe this article will help:
https://www.tutorialspoint.com/how-to-create-a-modal-dialog-in-tkinter

File: OptionScreen.py (hunk 4 above)
Comment (Nuzhyn-Ivan) [reply OptionScreen.py]: Now its better but still hardcoded:
self.languages = [Languages.current_lang["language_en"], Languages.current_lang["language_cz"]]

This list should be created automatically or at least get it from Languages.py

File: Languages.py [hunk 5]
```diff
@@ -1,117 +1,26 @@
...
-# configuration["registration_success"] = "Registrace proběhla úspěšně."
-# configuration["user_not_exist_warning"] = "Uživatel neexistuje!"
-# configuration["material_exist_warning"] = "Tento materiál již existuje."
-# configuration["material_specification_warning"] = "Nemůžete přidat materiál této specifikace.."
-# configuration["delete_material_warning"] = "Opravdu chcete odstranit tento materiál?"
-#
 
+def change_language(lang):
+    current_lang.clear()
+    config_object = ConfigParser()
```
Comment (Nuzhyn-Ivan) [inline Languages.py:22]: Do we really need to recreate ConfigParser for every change_language call?

**Pull request merged**